src/test/java/CreateAccountTest.java

```diff
@@ -21,6 +22,11 @@ public void setUp() {
        driver = WebDriverUtils.createWebDriver();
     }
 
+    @BeforeMethod
```

Comment:
Why before method is used? There is only one tests, so there is no need to use here.
Opening page should takes place in method annotated with @BeforeMethod.